src/App.tsx

```diff
@@ -15,36 +15,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
```

Comment:
better use const for enums because using `const enum` allows TypeScript to optimize the code by replacing symbolic enum names with their actual values during compilation. 

`const enum SortType`